docs/scratch_orgs.rst

```diff
@@ -1,38 +1,253 @@
++=============+==========================+=======================+==========+
+| ``dev``     | Development workflows    | ``orgs/dev.json``     | 7 days   |
++-------------+--------------------------+-----------------------+----------+
+| ``qa``      | Testing workflows        | ``orgs/dev.json``     | 7 days   |
++-------------+--------------------------+-----------------------+----------+
+| ``feature`` | Continuous Integration   | ``orgs/dev.json``     | 1 day    |
++-------------+--------------------------+-----------------------+----------+
+| ``beta``    | | Continuous Integration | ``orgs/beta.json``    | 1 day    |
+|             | | Hands-On Testing       |                       |          |
++-------------+--------------------------+-----------------------+----------+
+| ``release`` | | Continuous Integration | ``orgs/release.json`` | 1 day    |
+|             | | Hands-On Testing       |                       |          |
+|             | | Demo Workflow          |                       |          |
++-------------+--------------------------+-----------------------+----------+
+
+* ``dev`` is for development workflows and uses the ``orgs/dev.json`` configuration file. It has a seven-day lifespan.
+* ``qa`` is for testing workflows. ``qa`` and ``dev`` are the same out of the box, but can be customized to suit the needs of the project. It has a seven-day lifespan.
+* ``feature`` is for continuous integration and uses the ``orgs/dev.json`` configuration file. It has a one-day lifespan.
+* ``beta`` is for continuous integration or hands-on testing. It uses the ``orgs/beta.json`` configuration. It has a one-day lifespan.
+* ``release`` is for continuous integration, hands-on testing, or demo workflow. It uses the ``orgs/release.json`` configuration. It has a one-day lifespan.
+
+If your project has customized the available orgs, you'll see more orgs, or differences in configuration, compared to a new project.
+
+
+
+Create a Scratch Org
 --------------------
 
-Implicit creation of scratch orgs
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+To create a scratch org from a configuration, use it as the target of a command, task, or flow. CumulusCI automatically initializes orgs when they're first used.
+
+    Example: Create a scratch org from the ``dev`` configuration and review information about the created org.
+
+.. code-block:: console
+
+    $ cci org info dev
+
+..
+
+    When the org is created, it's associated with the name ``dev`` in the CumulusCI keychain and can be used with other commands until it expires. When an org expires or is removed, its associated configuration is left in place, and can be recreated whenever needed.
+
+It's possible to create new orgs in the keychain that inherit their configuration from a built-in org.
+
+    Example: Create a new org that uses the same configuration as type ``dev``, but is still independent of ``dev`` org.  
+    
+.. code-block ::
+
+    $ cci org scratch dev <org>
+    $ cci org info <org>
```

Comment:
The `<org>` variable name just doesn't sit well with me here. We use the word "org" a lot in this section, I'm inclined to go with `<org_name>` especially since we don't mention what the`<org>` parameter does in the command. I know that we haven't been doing that in other places in the doc, but I think it's worth being more precise as it's more informative to end users.

We could also go with a concrete example here:
```suggestion
.. code-block ::

    $ cci org scratch dev myDevOrg

You can verify that there is now an org with the name of ``myDevOrg`` that is associated with the ``orgs/dev.json`` configuration file by running ``cci org list``. 
```

Reply:
Okay. Changing all instances of <org> to <org_name>. But we still need to hash out whether or not we're doing this across the board.